app/controllers/avo/application_controller.rb

```diff
@@ -223,7 +223,7 @@ def resource
 
     def related_resource
       # Find the field from the parent resource
-      field = @resource.get_field params[:related_name]
+      field = @resource.get_field params[:related_name], for_view: :show
```

Comment:
We should not hard-code the view, it can also be `:edit`

Reply:
Makes sense. I'll follow your lead.